1.10/administering-clusters/deploying-a-local-dcos-universe.md

```diff
@@ -199,14 +199,28 @@ You can install and run DC/OS services on a datacenter without internet access w
     dcos node ssh --master-proxy --mesos-id=<mesos-id>
     ```
 
-1.  Use the following commands to download a copy of the DC/OS certificate locally and set it as trusted.
+1.  Use the following commands to download a copy of the DC/OS Local Universe certificate locally and set it as trusted for the docker daemon
```

Comment:
While we're here we should IMO say "DC/OS Local Universe CA certificate"

Reply:
yeah, ill take care of it and update it tomorrow.